app/src/main/java/com/cornellappdev/android/eatery/Model/MealType.java

```diff
@@ -0,0 +1,55 @@
+package com.cornellappdev.android.eatery.Model;
+
+import java.util.Comparator;
+
+/**
+ * Enum class to represent 4 different types of meal
+ */
+
+public enum MealType implements Comparable<MealType>{
+    BREAKFAST(1),
+    BRUNCH(2),
+    LUNCH(3),
+    DINNER(4);
+
+    private int index;
+
+    MealType(int index){
+        this.index = index;
+    }
+
+    public int getIndex() {
+        return index;
+    }
+
+    public static MealType fromDescription(String type) {
+        switch (type.toLowerCase()) {
+            case "breakfast":
+                return BREAKFAST;
+            case "brunch":
+                return BRUNCH;
+            case "lunch":
+                return LUNCH;
+            case "dinner":
+                return DINNER;
+        }
+        return null;
+    }
+}
+
+/**
+ * Sorts MealType in following order: Breakfast > Brunch > Lunch > Dinner
+ */
+
```

Comment:
Do you think we should add a comment about why we don't need to worry about comparing two meals of the same MealType?